### PR TITLE
New formula: sabaki 0.21.0

### PR DESCRIPTION
**Pull request by apetresc**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by gobinathm** [comment]
Fails

```
$ brew cask install https://raw.githubusercontent.com/apetresc/homebrew-cask/af4e2ab37766437995c6f7e16c3b9e2c91ef1f1a/Casks/sabaki.rb
==> Downloading https://raw.githubusercontent.com/apetresc/homebrew-cask/af4e2ab37766437995c6f7e16c3b9e2c91ef1f1a/Casks/sabaki.rb.
######################################################################## 100.0%
==> Satisfying dependencies
==> Installing Formula dependencies from Homebrew
unrar ... done
complete
==> Downloading https://github.com/yishn/Sabaki/releases/download/v0.21.0/sabaki-v0.21.0-mac-x64.7z
######################################################################## 100.0%
==> Verifying checksum for Cask sabaki
Error: Uh oh, could not figure out how to unpack '/Users/gobinathm/Library/Caches/Homebrew/Cask/sabaki--0.21.0.7z'
Error: nothing to install
```

**Comment by apetresc** [comment]
Huh, strange...

```
==> Satisfying dependencies
==> Installing Formula dependencies from Homebrew
unrar ... already installed
complete
==> Downloading https://github.com/yishn/Sabaki/releases/download/v0.21.0/sabaki-v0.21.0-mac-x64.7z
Already downloaded: /Users/apetresc/Library/Caches/Homebrew/Cask/sabaki--0.21.0.7z
==> Verifying checksum for Cask sabaki
==> Moving App 'Sabaki.app' to '/Applications/Sabaki.app'.
🍺  sabaki was successfully installed!
```

Looks like it's not figuring out how to do 7z... anyone have any idea why this might be?

**Comment by gobinathm** [comment]
You need to add depends_on after homepage

```
depends_on formula: 'unar'
```

**Comment by apetresc** [comment]
@gobinathm But that's been there since the beggining

**Comment by gobinathm** [comment]
works

```
$ brew cask install sabaki.rb
==> Satisfying dependencies
==> Installing Formula dependencies from Homebrew
unar ... done
complete
==> Downloading https://github.com/yishn/Sabaki/releases/download/v0.21.0/sabaki-v0.21.0-mac-x64.7z
Already downloaded: /Users/gobinathm/Library/Caches/Homebrew/Cask/sabaki--0.21.0.7z
==> Verifying checksum for Cask sabaki
==> Moving App 'Sabaki.app' to '/Applications/Sabaki.app'.
🍺  sabaki was successfully installed!
```


```
cask 'sabaki' do
  version '0.21.0'
  sha256 'e464ccb75867ca920b842a254231042489d9650a04cc782e2955b5ced4b086ea'

  # github.com was verified as official when first introduced to the cask
  url "https://github.com/yishn/Sabaki/releases/download/v#{version}/sabaki-v#{version}-mac-x64.7z"
  name 'Sabaki'
  homepage 'http://sabaki.yichuanshen.de/'

  depends_on formula: 'unar'

  license :mit

  app 'Sabaki.app'
end
```

**Comment by apetresc** [comment]
Wait, so you're saying on your machine, simply changing the order so the `depends_on` is _before_ the `license` is enough to make it work?

That makes no sense to me, but okay... that's an easy enough change :)

File: Casks/sabaki.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'sabaki' do
+  version '0.21.0'
+  sha256 'e464ccb75867ca920b842a254231042489d9650a04cc782e2955b5ced4b086ea'
+
+  # github.com/yishn/Sabaki was verified as official when first introduced to the cask
+  url "https://github.com/yishn/Sabaki/releases/download/v#{version}/sabaki-v#{version}-mac-x64.7z"
+  name 'Sabaki'
```
Comment (vitorgalvao) [inline Casks/sabaki.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
@apetresc Order of stanzas has no effect on execution order, the error was on `unrar` vs `unar`.

**Comment by apetresc** [comment]
@vitorgalvao @miccal Added appcast :) Audit etc still pass.

And thanks for pointing out the `unar` thing... I just consistently misread it as `unrar`, haha!

**Pull request merged**